TripPlanner/procedures/com/vantiq/trip/TripPlanner_tripActivate.vail

```diff
@@ -0,0 +1,29 @@
+// Activate a collaboration for the specified instance of the "trip" entity role.  Returns the id of the
+// active collaboration.
+PACKAGE com.vantiq.trip
+import type com.vantiq.trip.Trip
+
+STATELESS PROCEDURE TripPlanner.tripActivate(
+        entityId String REQUIRED DESCRIPTION "The id of the trip to be activated.", 
+        entity com.vantiq.trip.Trip REQUIRED DESCRIPTION "The trip instance to be stored in the active collaboration.", 
+        collaborationName String DESCRIPTION "The (optional) name to assign to the collaboration when created"
+    ): String
+
+// Set the collaboration name, if we have one
+var initialCollab = null
+if (collaborationName) {
+    initialCollab = {name: collaborationName}
+}
+
+// Activate a collaboration for the given entity (creating if necessary).
+var result = EXECUTE TripPlanner.ActiveCollabsGetOrCreateForEntity(entityId, "trip", entity, initialCollab) 
+    WITH partitionKey = entityId
+var collaborationId = result.collaboration.id
+
+// Check the value of "result.isNew" if you want to customize based on whether or not the collaboration
+// was created.
+
+// To start/resume a conversation for this collaboration, uncomment the following line:
+EXECUTE TripPlanner.ActiveCollabsStartConversation(collaborationId) WITH partitionKey = collaborationId
```

Comment:
this says to uncomment the following line, but the line is already uncommented.  So a little confusing...